### PR TITLE
Add 2018 Visits

### PR DESCRIPTION
**Pull request by KerryHite**
Technical edit

### CODE REVIEW

**Comment by KerryHite** [comment]
In progress. @joewiz I will tag you when ready for your review.

**Comment by KerryHite** [comment]
@joewiz Ready for you to review

File: data/2010.xml [hunk 1]
```diff
@@ -5176,4 +5176,244 @@
...
+        <created-by>wicentowskijc</created-by>
+        <created-datetime>2019-07-25T16:12:57.84-04:00</created-datetime>
+        <last-modified-by>wicentowskijc</last-modified-by>
+        <last-modified-datetime>2019-07-25T16:12:57.84-04:00</last-modified-datetime>
+    </visit>
+    <visit>
+        <id>2912</id>
+        <start-date>2018-05-22</start-date>
+        <end-date>2018-05-22</end-date>
+        <visitor>President Moon-Jae-in</visitor>
```
Comment (joewiz) [inline data/2010.xml]: @KerryHite I just noticed the dash between the surname and given name here. (Clicking on the link in the email notification will take you right to the line I'm referring to on the GitHub website.)

Comment (KerryHite) [reply data/2010.xml]: @joewiz thanks for the catch. Corrected 2017 visit also. Ready to go.

Comment (joewiz) [reply data/2010.xml]: @KerryHite Thank you!

**Pull request merged**